content/python/concepts/random-module/terms/triangular/triangular.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'triangular'
+Description: 'Returns a random floating number from a triangular distribution.'
+Subjects: 
+  -"Computer Science"
+  -"Data Science"
+Tags:
+  -"Functions"
+  -"Methods"
+  -"Random"
+Catalog Content: 
+  -"learn-python-3"
+  -"paths/computer-science"
+  -"paths/data-science"
+---
```

Comment:
```suggestion
---
Title: '.triangular()'
Description: 'Returns a random floating number from a triangular distribution.'
Subjects: 
  -'Computer Science'
  -'Data Science'
Tags:
  -'Functions'
  -'Methods'
  -'Random'
Catalog Content: 
  -'learn-python-3'
  -'paths/computer-science'
  -"paths/data-science'
---
```

Reply:
Meta data to be written in single quotes ( ' ' ) and the title should be the term itself.